docs/dsp-tools-create.md

```diff
@@ -1,15 +1,17 @@
+[![PyPI version](https://badge.fury.io/py/dsp-tools.svg)](https://badge.fury.io/py/dsp-tools)
+
 # JSON data model definition format
 
 ## Introduction
-This document contains all the information you need to create an data model that's used by DSP. According to
-Wikipedia, da [data model](https://en.wikipedia.org/wiki/Data_model) is "_is an abstract model that organizes elements
+This document contains all the information you need to create a data model that can be used by DSP. According to
+Wikipedia, the [data model](https://en.wikipedia.org/wiki/Data_model) is "_an abstract model that organizes elements
 of data and standardizes how they relate to one another and to the properties of real-world entities._"  Further it
 states: "_A data model explicitly determines the structure of data. Data models are typically specified by a data
-specialist, data librarian, or a digital humanities scholar in a data modeling notation_". In this section we will 
-describe one of the notations that is used by the _dsp-tools_ to create a data model in the dsp repository. The dsp
-repository is loosely based on [Linked Open Data](https://en.wikipedia.org/wiki/Linked_data) where also the term
-_"ontology"_ is used for the data model. It should be noted that in this context an ontology is not used in the
-philosophical sense.
+specialist, data librarian, or a digital humanities scholar in a data modeling notation_".
+
+In this section, we will describe one of the notations that is used by dsp-tools to create a data model in the DSP
```

Comment:
Personally, I'm not overly fond of those "we" clauses. To me that sounds too narrative and less instruction-like than e.g. a passive construction. (But that's cosmetics)

Reply:
I totally agree with you. But the whole file is written like this (and in general, in a very informal/spoken style). I leave it as is for the moment. Otherwise we would have to rewrite all the files from scratch.